api/nakadi-event-bus-api.yaml

```diff
@@ -1132,6 +1161,13 @@ definitions:
           If set by the producer Event will be rejected.
         format: date-time
         example: '1996-12-19T16:39:57-08:00'
+      version:
```

Comment:
Sorry, forget about my comment here. This is event metadata it is not supposed to be submitted by user anyway.